tests/static_routes_test.py

```diff
@@ -0,0 +1,100 @@
+import os
+import traceback
+from unittest import mock
+
+from click.testing import CliRunner
+
+import config.main as config
+import show.main as show
+from .utils import get_result_and_return_code
+from utilities_common.db import Db
+
+ROUTE_UPDATE_STR = '''
+Added static route 1.2.3.4/32
+Added static route 2.2.3.4/32
+Added static route 3.2.3.4/32
+''' 
+
+
+class TestStaticRoutes(object):
+    @classmethod
+    def setup_class(cls):
+        os.environ['UTILITIES_UNIT_TESTING'] = "1"
+        print("SETUP")
+
+    '''Add'''
+    def test_add_simple_static_route(self):
+        db = Db()
+        runner = CliRunner()
+        obj = {'config_db':db.cfgdb}
+
+        # config route add prefix 1.2.3.4/32 nexthop 30.0.0.5
+        result = runner.invoke(config.config.commands["route"].commands["add"], \
+        ["prefix", "1.2.3.4/32", "nexthop", "30.0.0.5"], obj=obj)
+        print(result.exit_code, result.output)
+        assert ('1.2.3.4/32') in db.cfgdb.get_table('STATIC_ROUTE')
+        assert db.cfgdb.get_entry('STATIC_ROUTE', '1.2.3.4/32') == {"nexthop": "30.0.0.5"}
+
+    def test_add_vrf_static_route(self):
+        db = Db()
+        runner = CliRunner()
+        obj = {'config_db':db.cfgdb}
+
+        # config route add prefix 2.2.3.4/32 nexthop 30.0.0.6
+        result = runner.invoke(config.config.commands["route"].commands["add"], \
+        ["prefix", "vrf", "Vrfblue", "2.2.3.4/32", "nexthop", "30.0.0.6"], obj=obj)
+        print(result.exit_code, result.output)
+        assert ('2.2.3.4/32') in db.cfgdb.get_table('STATIC_ROUTE')
+        assert db.cfgdb.get_entry('STATIC_ROUTE', '2.2.3.4/32') == {"nexthop": "30.0.0.6", "vrf_name": "Vrfblue"}
+
+    def test_add_dest_vrf_static_route(self):
+        db = Db()
+        runner = CliRunner()
+        obj = {'config_db':db.cfgdb}
+
+        # config route add prefix 3.2.3.4/32 nexthop 30.0.0.6
+        result = runner.invoke(config.config.commands["route"].commands["add"], \
+        ["prefix", "3.2.3.4/32", "nexthop", "vrf", "Vrfred", "30.0.0.6"], obj=obj)
+        print(result.exit_code, result.output)
+        assert ('3.2.3.4/32') in db.cfgdb.get_table('STATIC_ROUTE')
+        assert db.cfgdb.get_entry('STATIC_ROUTE', '3.2.3.4/32') == {"nexthop": "30.0.0.6", "nexthop_vrf": "Vrfred"}
```

Comment:
I think we need to follow a certain schema design for the STATIC_ROUTE table. In https://github.com/Azure/SONiC/pull/585, there is a schema for the table but it seems different here.